catalog.html

```diff
@@ -33,37 +33,37 @@ <h2 class="visually-hidden">Фильтр шаблонов</h2>
             <h2 class="visually-hidden">Список шаблонов</h2>
             <ul class="catalog-list">
                 <li class="catalog-item">
-                    <img src="img/sedona.jpg" width="220" height="165" alt="Шаблон Седона">
+                    <img src="img/img-sedona.jpg" width="360" height="576" alt="Шаблон Седона">
```

Comment:
```suggestion
                    <img src="img/sedona.jpg" width="360" height="576" alt="Шаблон Седона">
```
Приставка `img` не нужна и так понятно что это картинка